src/libostree/ostree-core.c

```diff
@@ -452,15 +444,92 @@ ostree_raw_file_to_content_stream (GInputStream       *input,
   g_ptr_array_add (streams, g_object_ref (header_in_stream));
   if (input)
     g_ptr_array_add (streams, g_object_ref (input));
-  
-  ret_input = (GInputStream*)ostree_chain_input_stream_new (streams);
 
-  ret = TRUE;
+  ret_input = (GInputStream*)ostree_chain_input_stream_new (streams);
   ot_transfer_out_value (out_input, &ret_input);
+  if (out_header_size)
+    *out_header_size = header_size;
+
+  return TRUE;
+}
+
+/**
+ * ostree_raw_file_to_archive_z2_stream:
+ * @input: File raw content stream
+ * @file_info: A file info
+ * @xattrs: (allow-none): Optional extended attributes
+ * @out_input: (out): Serialized object stream
+ * @out_length: (out): Length of stream
```

Comment:
Uh, this is addressed, but apparently github moved your comment to the `ostree_raw_file_to_content_stream` function docs…